core/src/components/datetime/datetime.tsx

```diff
@@ -459,27 +480,41 @@ export class Datetime implements ComponentInterface {
    */
   @Method()
   async confirm(closeOverlay = false) {
+    const { highlightActiveParts, isCalendarPicker, activeParts } = this;
+
     /**
      * We only update the value if the presentation is not a calendar picker,
      * or if `highlightActiveParts` is true; indicating that the user
      * has selected a date from the calendar picker.
      *
      * Otherwise "today" would accidentally be set as the value.
      */
-    if (this.highlightActiveParts || !this.isCalendarPicker) {
-      /**
-       * Prevent convertDataToISO from doing any
-       * kind of transformation based on timezone
-       * This cancels out any change it attempts to make
-       *
-       * Important: Take the timezone offset based on
-       * the date that is currently selected, otherwise
-       * there can be 1 hr difference when dealing w/ DST
-       */
-      const date = new Date(convertDataToISO(this.activeParts));
-      this.activeParts.tzOffset = date.getTimezoneOffset() * -1;
+    if (highlightActiveParts || !isCalendarPicker) {
+      const activePartsIsArray = Array.isArray(activeParts);
+      if (activePartsIsArray && activeParts.length === 0) {
+        this.value = undefined;
```

Comment:
What is the reasoning for setting value to undefined here?

Reply:
If no dates are selected, the value is set to `undefined` per the design doc: https://www.notion.so/ionic/ion-datetime-multiple-date-selection-63c95d300d2a42a79d956b3c4e3244b1#e80ea660462541c2a2e8a51a1ab157b9 The intention was to keep in parity with a single-selection datetime that has nothing selected due to not being inited with a default value.

Reply:
Ahh I see now. If you haven't selected a date and you click to confirm, the value should be emitted as `undefined`, instead of trying to convert the active parts into an ISO date string. 

This makes sense, I missed that part originally 👍